docs/SOPs/Managed_access/Review_and_export_SOP.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Data and metadata review and export SOP
+parent: Managed access
+grand_parent: SOPs
+nav_order: 5
+last_modified_date: 30/05/2024
+---
+
+<script src="https://kit.fontawesome.com/fc66878563.js" crossorigin="anonymous"></script>
+
+# Managed access dataset - Data and metadata review and export SOP
+Once the the wrangler has verified that the data and metadata files were transferred correctly to the hca-util are they can  proceed with submission creation and validation
+
+## Submission review and validation
+Data and metadata in a managed access submission are considered sensitive information and must be kept encrypted. Unlike in the case of open access datasets, review for a managed access dataset must be done without ever downloading the metadata spreadsheet - or the data - to a non-encrypted location, like a laptop.
```

Comment:
Data and metadata for a managed access project submission are considered sensitive information and must be kept encrypted at all times. Unlike open access projects, data wrangler review on a managed access project must be done without downloading the metadata spreadsheet or the data to a non-encrypted location, like a laptop.